agb/src/input.rs

```diff
@@ -285,6 +285,11 @@ impl ButtonController {
     /// }
     /// # }
     /// ```
+    ///
+    /// ## Bug
+    ///
+    /// This has confusing behaviour if you pass more than one button to `keys` (via the `|` operator). This will be
```

Comment:
What's confusing about this behavior? I agree that _just pressing_ two buttons in the same frame is highly unlikely. Maybe it should say something like this.

```suggestion
    /// This method is not intended to be used with more than one button (via the `|` operator). This will be
```

Are there people who use `union` I think that method has the same behavior for `Button` as the `|` operator.